src/styles/_catalog.scss

```diff
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform 0.3s color 0.3s;
```

Comment:
Transition duration should be variable, you have that more than once in your code.

Reply:
ok , i did it)